### ISSUE TITLE
After Adding POI, It Doesn't Appear on Map and Isn't Searchable

### ISSUE THREAD

**Issue by seniorm0ment** [osmandapp/OsmAnd#16451]
If I pick a building, hold down and click actions, create POI, create the POI, then upload it. It doesn't show up after creating it. Also tried updating the map in settings, did not find it still.

**Comment by Zirochkabila** [comment]
@seniorm0ment Please clarify:
1. The device you use, OS version
2. Application version
3. A short video and screenshots where we could see the error
4. Logcat file: Menu > Help > Send logcat log

**Comment by scaidermern** [comment]
Probably a duplicate of #2890.

Only with OsmAnd live uploaded POIs will become visible (but only after 1-2 hours). Without OsmAnd live you have to wait for a new base map update, which will happen at the start of the next month.

**Comment by vshcherb** [comment]
It's a duplicste

**Comment by vshcherb** [comment]
OSM Live doesn't support address updates (another duplicate)
